python/python/lancedb/rerankers/base.py

```diff
@@ -130,12 +134,91 @@ def merge_results(self, vector_results: pa.Table, fts_results: pa.Table):
         combined = pa.concat_tables(
             [vector_results, fts_results], **self._concat_tables_args
         )
-        row_id = combined.column("_rowid")
 
         # deduplicate
-        mask = np.full((combined.shape[0]), False)
+        combined = self._deduplicate(combined)
+
+        return combined
+
+    def rerank_multivector(
+        self,
+        vector_results: Union[List[pa.Table], List["LanceVectorQueryBuilder"]],
+        query: Union[str, None] = None,  # Some rerankers might not need the query
+        deduplicate: bool = False,
+        # deduplicator: callable = None,
+    ):
+        """
+        This is a rerank function that receives the results from multiple
+        vector searches. Could be using different vector columns
+
```

Comment:
```suggestion
        This is a rerank function that receives the results from multiple
        vector searches. For example, this can be used to combine the
        results of two vector searches with different embeddings.

```